kuma/wiki/models.py

```diff
@@ -1358,7 +1358,7 @@ def parents(self):
             current_parent = current_parent.parent_topic
         return parents
 
-    def get_other_translations(self, fields=[]):
+    def get_other_translations(self, fields=()):
```

Comment:
This is just force of habit in avoiding mutable types as keyword defaults. It's not required for this PR.